lib/router/transition.js

```diff
@@ -325,16 +326,11 @@ Transition.prototype.send = Transition.prototype.trigger;
 /**
   @private
 
-  Logs and returns a TransitionAborted error.
+  Logs and returns an instance of TransitionAbortedError.
  */
 function logAbort(transition) {
   log(transition.router, transition.sequence, "detected abort.");
-  return new TransitionAborted();
+  return new TransitionAbortedError();
 }
 
-function TransitionAborted(message) {
-  this.message = (message || "TransitionAborted");
-  this.name = "TransitionAborted";
-}
-
-export { Transition, logAbort, TransitionAborted };
+export { Transition, logAbort };
```

Comment:
We should probably export `TransitionAbortedError` at `TransitionAborted` for back-compat reasons.